releng/tag-release.sh

```diff
@@ -113,7 +113,13 @@ if ! (head -n 1 "${relnotes}" | grep -q "ZFSBootMenu ${tag}\b"); then
 fi
 
 # Update version in generate-zbm
-sed -i bin/generate-zbm -e "s/our \$VERSION.*/our \$VERSION = '${release}';/"
+if [ -x releng/version.sh ]; then
+  error "ERROR: unable to update release version"
+fi
+
+if ! out="$( releng/version.sh )"; then
```

Comment:
Should this be called with `-u` so that versions are updated?

Reply:
It should also be called with `-v` to set the version as we expect. Fixed.